pyproject.toml

```diff
@@ -265,6 +265,7 @@ markers = [
 # We disable testing for the following wheels:
 # - MuslLinux (tests hang non-deterministically)
 test-skip = "*-musllinux_x86_64"
+test-sources = ["pyproject.toml", "conftest.py"]
```

Comment:
```suggestion
test-sources = ["README.md"]
```

Reply:
If it is the case you don't need anything at all copied in, you can actually skip test-sources now on iOS, so if this passes, I'd say you don't need it at all.